cfme/infrastructure/datastore.py

```diff
@@ -124,17 +135,14 @@ def get_hosts(self):
         if not self._on_hosts_page():
             sel.force_navigate('infrastructure_datastore', context=self._get_context())
             try:
-                list_acc.select('Relationships', 'Hosts', by_title=False, partial=True)
+                sel.click(details_page.infoblock.element("Relationships", "Hosts"))
```

Comment:
Presumably this is to get around the fact that the relationships links disappeared. This is already in another PR IIRC.